test/Generator/ClassGeneratorTest.php

```diff
@@ -1105,6 +1105,17 @@ public function testCorrectExtendNames()
         $classGenerator->setName('ClassName');
         $classGenerator->setExtendedClass('DateTime');
         $this->assertContains('class ClassName extends DateTime', $classGenerator->generate());
+
+        $classGenerator = new ClassGenerator();
+        $classGenerator->setName('ClassName');
+        $classGenerator->setNamespaceName('SomeNamespace');
+        $classGenerator->setExtendedClass('\DateTime');
+        $this->assertContains('class ClassName extends \DateTime', $classGenerator->generate());
+
+        $classGenerator = new ClassGenerator();
+        $classGenerator->setName('ClassName');
+        $classGenerator->setExtendedClass('\DateTime');
+        $this->assertContains('class ClassName extends DateTime', $classGenerator->generate());
```

Comment:
Please create new test methods for these two new cases, using a name descriptive of the behavior being tested, and annotate them with `@group 75`.

Thanks!